e2e/testdata/fn-render/inline-fnconfig/.expected/diff.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/resources.yaml b/resources.yaml
+index 10cd18b..c77c357 100644
+--- a/resources.yaml
++++ b/resources.yaml
+@@ -11,17 +11,29 @@
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-
```

Comment:
We should avoid this kind of format changing in the tests. https://github.com/GoogleContainerTools/kpt/pull/1663#discussion_r605989934

Reply:
Agreed. Done.